engines/director/lingo/lingo-builtins.cpp

```diff
@@ -3663,9 +3663,24 @@ void LB::b_scrollByPage(int nargs) {
 }
 
 void LB::b_lineHeight(int nargs) {
-	g_lingo->printSTUBWithArglist("b_lineHeight", nargs);
-	g_lingo->dropStack(nargs);
-	Datum res(1);
+	Datum lineNum = g_lingo->pop();
+	Datum castRef = g_lingo->pop();
+
+	if (castRef.type != CASTREF) {
+		g_lingo->lingoError("Incorrect argument type for lineHeight");
+        return;
```

Comment:
I was not  clear here. You _must_ return something, so the execution continues in case we want to ignore Lingo errors. So, you must push here `1` on the stack. Same for the error below.